mentor-guidebook/sample-answers/week8/controllers/taskController.js

```diff
@@ -0,0 +1,326 @@
+const prisma = require('../prisma/db');
+const { taskSchema, patchTaskSchema } = require("../validation/taskSchema");
+
+exports.index = async (req, res) => {
```

Comment:
In my examples in the text, I used a different style: module.exports = {index, show, etc}.  The style you are following, 
exports.index =
is maybe better, but we have to make sure to describe both styles in the basics lesson (week 1 or 2?).